kvmagent/kvmagent/plugins/tensorfusion/executor.py

```
@@ -0,0 +1,597 @@
+        try:
+            import psutil
+            p = psutil.Process(worker.pid)
+            return p.is_running() and p.status() != psutil.STATUS_ZOMBIE
+        except ImportError:
+            # Fallback: use kill(0), then check Linux /proc state to filter zombies.
+            try:
+                os.kill(worker.pid, 0)
+                return not ProcessExecutor._is_linux_zombie(worker.pid)
+            except OSError:
+                return False
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)